pyproject.toml

```diff
@@ -11,6 +11,8 @@ python-dotenv = "^0.14.0"
 requests = "^2.24.0"
 pytest = "^6.1.2"
 gunicorn = "^20.0.4"
+selenium = "^3.141.0"
+webdriver-manager = "^3.3.0"
```

Comment:
pytest, selenium, webdriver-manager should be in the dev-dependencies section so that you can run `poetry install --no-dev` to skip them in production